src/shared/components/oncoprint/controls/styles.scss

```diff
@@ -37,6 +36,10 @@
     }
   }
 
+  .dropdown-menu.heatmap {
+    width:450px;
```

Comment:
would max-width work and still allow it to shrink to size when necessary?

Reply:
No max-width does not work. Width of the menu is not affected by the menu entries. Can you live with 450px fixed?